### DISCUSSION TITLE
os.kill(pid, signal.SIGTERM)，tips:  _signal_handler() takes 1 positional argument but 3 were given

### DISCUSSION THREAD

**Discussion by coolerwu** [fastapi/fastapi#11546] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
os.kill(pid, signal.SIGTERM)
```

### Description

tips:  _signal_handler() takes 1 positional argument but 3 were given

### Operating System

Linux

### FastAPI Version

0.111.0

### Pydantic Version

2.7.1

### Python Version

3.8

**Comment by JavierSanchezCastro** [comment, marked as answer]
Is this a FastAPI issue?

**Comment by coolerwu** [comment]
sry
